test/hyrdate-test.js

```diff
@@ -0,0 +1,49 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const User = require('../src/user');
+const Hydration = require('../src/hyrdate');
+
+describe('Hydration', function() {
+
+  let hydration, hydrationData;
+
+  beforeEach(function() {
+    hydrationData = [
+      {
+          "userID": 1,
+          "date": "2019/06/15",
+          "numOunces": 37
+        },
+      {
+        "userID": 1,
+        "date": "2019/06/16",
+        "numOunces": 69
+      },
+        {
+        "userID": 2,
+        "date": "2019/06/17",
+        "numOunces": 96
+      }
+    ];
+
+      hydration = new Hydration(hydrationData);
+  });
+
+
+  it('should be a function', function() {
+    expect(Hydration).to.be.a('function');
+  })
+
+  it('should be an instance of Hydration', function() {
```

Comment:
The three tests for each of the methods will also need to be changed to reflect what the methods should be returning.  There is also no test currently in here that tests the returnWeeklyConsumption function.